ci/constants.py

```diff
@@ -96,9 +97,15 @@ class TargetPython(Enum):
     'sympy',
     'vlc',
 ])
+BROKEN_RECIPES_PYTHON2 = set([
+])
+# to be created via https://github.com/kivy/python-for-android/issues/1514
+BROKEN_RECIPES_PYTHON3 = set([
```

Comment:
This list is already defined here https://github.com/kivy/python-for-android/blob/0563ef00ab5a64184f1290941ff2d8ee564cacbc/ci/constants.py#L70 why would we override?
I would keep that comment about #1514 however, good idea